bugsnag-android-core/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -259,6 +265,21 @@ public void onOrientationChanged(int orientation) {
         loadPlugins();
     }
 
+    void recordStorageCacheBehavior(MetaData metaData) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            File cacheDir = appContext.getCacheDir();
+
+            try {
+                boolean tombstone = storageManager.isCacheBehaviorTombstone(cacheDir);
```

Comment:
Can this in any way be deceptively false, e.g. if `storageManager.isCacheBehaviorTombstone(cacheDir) == false` but `storageManager.isCacheBehaviorTombstone(cacheDir + "/bugsnag-errors") == true`?

Reply:
That's a possibility - although it would require for a user to have specifically called `storageManager.setCacheBehaviorTombstone` on the `bugsnag-errors` directory.

Reply:
Is there anything preventing checking the bugsnag-errors directory itself to avoid misleading values?

Reply:
not that I'm aware of - I plan on updating the PR to do this.